apps/control/src/ui/lowerthirds/TickerState.lt.ui.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Select, SelectItem, Switch, Text } from "@mantine/core"
+import { useInputState, useToggle } from "@mantine/hooks"
+import { Live, LowerthirdData } from "interface/ws"
+import { useLt } from "utils/hooks"
+import { useBSave } from "../../context/bsave/bsave.hook"
+import { setLive } from "utils/socket/events"
+import { InputHTMLAttributes, useState } from "react"
+
+const TickerState = () => {
+  const { mode, show, data } = useLt()
+  const bSave = useBSave()
+
+  const toggleLowerThirds = () => {
+    const toggleLT: Partial<Live> = {
+      lt: { show: !show, data, mode },
+    }
+    setLive(toggleLT)
+    bSave(toggleLT)
+  }
+
+  const onModeSelect = (modeValue: any) => {
+    const changeMode: Partial<Live> = {
+      lt: { mode: modeValue, data, show },
+    }
+    setLive(changeMode)
+    bSave(changeMode)
+  }
```

Comment:
Always ensure the type and add a fallback value

```suggestion
  const onModeSelect = (modeValue: keyof LowerthirdData | null) => {
    const changeMode: Partial<Live> = {
      lt: { mode: modeValue || "ticker", data, show },
    }
    setLive(changeMode)
    bSave(changeMode)
  }
```